### PR TITLE
tests: stdlib: Add unit tests for strpbrk()

### LINKED ISSUE #1848: Add tests for strpbrk()
Implement some test cases for [standard strpbrk()](https://pubs.opengroup.org/onlinepubs/009695399/functions/strpbrk.html)

### PR DESCRIPTION
**Pull request by nradk**
This fixes #1848.

### CODE REVIEW

**Comment by anton-bondarev** [comment]
@nradk I've approved it. But there are some conflicts.
Can you rebase your branch under embox/master (upstream/master) and solve the conflicts

And don't forget to load your final proposal to GSoC site

File: src/tests/stdlib/strpbrk.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	test_assert_equal(strpbrk(embox, embox), embox);
+	test_assert_equal(strpbrk(embox, vowels_lower), embox + 5);
+	test_assert_equal(strpbrk(vowels_lower, embox), vowels_lower + 1);
+	test_assert_equal(strpbrk(digits, nine), digits + 9);
+	test_assert_null(strpbrk(symbols, nine));
+	test_assert_equal(strpbrk(symbols, hash), symbols + 3);
+}
+
+TEST_CASE("strpbrk with non-printable data") {
+	char data[] = {0xCA, 0xFE, 0xBA, 0xBE, 0x00};
```
Comment (anton-bondarev) [inline src/tests/stdlib/strpbrk.c:40]: maybe escape sequence is used here ([https://en.wikipedia.org/wiki/Escape_sequences_in_C](https://en.wikipedia.org/wiki/Escape_sequences_in_C))

File: templates/x86/test/units/mods.config [hunk 2]
```diff
@@ -84,6 +84,7 @@ configuration conf {
 	@Runlevel(1) include embox.test.stdlib.strcmp_test
 	@Runlevel(1) include embox.test.stdlib.strlwr_test	
 	@Runlevel(1) include embox.test.stdlib.strcasestr_test	
+	@Runlevel(1) include embox.test.stdlib.strpbrk_test
```
Comment (anton-bondarev) [inline templates/x86/test/units/mods.config:87]: Can you split commit for two? one with code and other with adding to template?

File: src/tests/stdlib/strpbrk.c (hunk 1 above)
Comment (nradk) [reply src/tests/stdlib/strpbrk.c]: Used escape sequences inside quotation marks to encode non-printable strings.

File: templates/x86/test/units/mods.config (hunk 2 above)
Comment (nradk) [reply templates/x86/test/units/mods.config:87]: Split the commit as you asked.

**Comment by anton-bondarev** [comment]
@nradk Thanks a lot!
I've merged it.

**Pull request merged**